pkg/synopsysctl/cmd_start.go

```diff
@@ -29,20 +29,20 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// startCmd starts a resource in the cluster if it's stopped
+// startCmd put a resource into the start state
```

Comment:
I don't think there's a need to make these textual changes, an in fact it's more confusing imo.  start and stop were properly described before these changes.  The commands start or stop a resource.

Reply:
@rrati I was trying to make it consistent with db-migrate command because they are all changing the state of the resource, but I can see how that's an implementation detail so I will change them back.